sdk/cosmos/azure-cosmos/test/test_container_properties_cache.py

```diff
@@ -71,7 +73,372 @@ def test_container_properties_cache(self):
         # Now we can compare the RID and Partition Key Definition
         self.assertEqual(cached_properties.get("_rid"), fresh_container_read.get("_rid"))
         self.assertEqual(cached_properties.get("partitionKey"), fresh_container_read.get("partitionKey"))
-        client.delete_database(created_db)
+        created_db.delete_container(container_name)
+
+    def test_container_recreate_create_upsert_replace_item(self):
```

Comment:
For the sake of completeness - how about we test query, change feed pull model, batch?

Reply:
Thank you for the suggestion, I have added tests to cover all three of them as well.